inventory-repository/src/main/resources/ldap/inventory.schema

```diff
@@ -821,42 +821,47 @@ objectclass ( InventoryObjectClasses:3
   SUP top
   AUXILIARY )
 
-objectclass ( InventoryObjectClasses:80:1
+objectclass ( InventoryObjectClasses:80.1
```

Comment:
won't we need a migration script for that ? existing entries will not agree with a change of objectclass